crates/pop-parachains/src/templates.rs

```diff
@@ -154,6 +166,30 @@ pub enum Parachain {
 		)
 	)]
 	ParityFPT,
+	/// Tanssi generic container chain template.
+	#[strum(
+		serialize = "simple",
+		message = "Simple",
+		detailed_message = "Generic container chain template.",
+		props(
+			Provider = "Tanssi",
+			Repository = "https://github.com/moondance-labs/tanssi",
+			Network = "./network.toml"
+		)
+	)]
+	TanssiSimple,
+	/// Tanssi EVM enabled container chain template.
+	#[strum(
+		serialize = "frontier",
+		message = "EVM",
+		detailed_message = "EVM enabled container chain template.",
```

Comment:
https://github.com/moondance-labs/tanssi/blob/1fe3bc169163b778e3b4251698de22131b60c985/container-chains/runtime-templates/frontier/Cargo.toml#L4 might be sligthly nicer, only as its something they have defined rather than us.